backend/audit/intakelib/checks/util.py

```diff
@@ -60,3 +65,38 @@ def get_names_of_all_ranges(data):
                 if "name" in range_item:
                     names.append(range_item["name"])
     return names
+
```

Comment:
Due to the code below, I am planning to move `util.py` and `error_messages`  from `checks` to `intakelib` so that code from both `checks` and `transforms` can reuse these fonctions when needed. Waiting on #2553 to get merged to main first.